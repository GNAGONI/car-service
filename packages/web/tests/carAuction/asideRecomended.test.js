import React from 'react'
import renderer from 'react-test-renderer'
import AsideRecomended from '../../components/carAuction/asideRecomended'

it('renders correctly', () => {
  const tree = renderer.create(<AsideRecomended />).toJSON()
  expect(tree).toMatchSnapshot()
})
