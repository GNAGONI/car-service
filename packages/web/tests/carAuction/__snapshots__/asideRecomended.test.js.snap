// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<aside
  className="aside-recommend text-default"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="text-block col-md-8"
        lb={9}
      >
        <div
          className="text-box"
        >
          <h2>
            Advertise your car to over 
            <strong>
              16,000 garages across the UK
            </strong>
          </h2>
        </div>
        <div
          className="btn-block"
        >
          <a
            href="/car-auctions/auction-your-car"
            onClick={[Function]}
          >
            Auction your car
          </a>
        </div>
      </div>
      <div
        className="d-none d-md-flex col-md-3"
      >
        <div
          className="img-area"
        >
          <img
            alt="Would you recommend your car?"
            height="153"
            src=""
            width="143"
          />
        </div>
      </div>
    </div>
  </div>
</aside>
`;
